fastlane/Fastfile

```diff
@@ -0,0 +1,286 @@
+require 'yaml'
+require_relative 'version'
+
+desc "Commit version file"
+private_lane :github_commit_version do |options|
+  require 'openssl'
+  require 'jwt'
+  require 'octokit'
+  require 'base64'
+
+  branch = git_branch() # Commit to current branch
+  version_yml_path = options[:version_yml_path]
+  repo = options[:repo]
+  new_version_code = options[:new_version_code]
+  new_version_name = options[:new_version_name]
+  app_id = options[:app_id]
+  private_key = OpenSSL::PKey::RSA.new(Base64.decode64(options[:private_key]))
+
+  payload = {
+      iat: Time.now.to_i,
+      exp: Time.now.to_i + (5 * 60),
+      iss: app_id
+  }
+
+  jwt = JWT.encode(payload, private_key, 'RS256')
+  client = Octokit::Client.new(:bearer_token => jwt)
+
+  installation = client.find_repository_installation(repo)
+  token = client.create_app_installation_access_token(installation.id).token
+  client = Octokit::Client.new(:access_token => token)
+
+  version_yml_content = client.contents(repo, path: version_yml_path, query: {ref: branch})
+  version_yml = YAML.load(Base64.decode64(version_yml_content.content))
```

Comment:
Why do you load the version from the origin git repo at all?
You could just as well create `version_yml` as a new object attaching `new_version_code` and `new_version_name` as attributes, right?

Reply:
This action can also be called locally. I did not want to depend on local git tools and the state of the version.yml. The action should bump the version in git without side-effects.